JK Hera/src/main/java/ayds/hera/songinfo/moredetails/model/repository/broker/ProxyWikipedia.kt

```diff
@@ -0,0 +1,35 @@
+package ayds.hera.songinfo.moredetails.model.repository.broker
+
+import ayds.hera.songinfo.moredetails.model.entities.*
+import ayds.zeus1.wikipedia.WikipediaService
+import ayds.zeus1.wikipedia.entities.Article
+import java.lang.Exception
+
+internal class ProxyWikipedia(
+    private val wikipediaService : WikipediaService
+): Proxy {
+
+    override fun getCard(artistName: String): Card? {
+        var serviceWikipedia: Article? = null
+        try {
+            serviceWikipedia = wikipediaService.getArticle(artistName)
+        }catch (e: Exception) {
+            println("Exception from WikipediaData")
+        }
+
+        var card: Card? = null
+        serviceWikipedia?.let {
+            card = mapArticleToCard(it)
+        }
+
+        return card
```

Comment:
Se puede simplificar:
```suggestion
       return  try {
            wikipediaService.getArticle(artistName)?.let {
                 mapArticleToCard(it)
            }
        }catch (e: Exception) {
            println("Exception from WikipediaData")
            null
        }
```